src/Og.php

```diff
@@ -269,6 +269,34 @@ public static function isGroupContent($entity_type_id, $bundle_id) {
     return \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity_type_id, $bundle_id);
   }
 
+  /**
+   * Gets the Group entities where an entity belongs.
+   *
+   * This works by checking if the bundle has at least one group audience field
+   * filled.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity object.
+   *
+   * @return array
+   *   List of groups where the entity belongs.
+   */
+  public static function getEntityGroups(EntityInterface $entity) {
+    $can_be_group_content = \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity->getEntityTypeId(), $entity->bundle());
+    if (!$can_be_group_content) {
+      return FALSE;
```

Comment:
I think it should be either an empty array - but maybe even better throw an Exception?

Not sure, which one would be better. @pfrenssen any thoughts?